implementations/rust/ockam/ockam_command/src/output/output.rs

```diff
@@ -434,15 +436,15 @@ impl fmt::Display for X25519PublicKeyDisplay {
     }
 }
 
-pub struct Ed25519PublicKeyDisplay(pub Ed25519PublicKey);
+pub struct Ed25519PublicKeyDisplay(pub EdDSACurve25519PublicKey);
```

Comment:
I would like to avoid pulling Display related things to `ockam_identity` crate, I think it's `ockam_command` specific how we would like to print it. Therefore, introducing new type is the way to imply this behaviour on the type from another crate

Reply:
I see. I also think that displaying data for `command` should have even its own type class.